src/js/dashboard/dashboard.controller.js

```diff
@@ -1,9 +1,76 @@
 'use strict';
 
-function DashboardComponent() {
+function DashboardComponent(franklinAPIService, $scope, $location,
+  $auth, toastr, ENV, $httpParamSerializer, $state, $modal) {
   /* jshint validthis: true */
   let dc = this;
-  
+  dc.username = "";
+  dc.deployedRepos = [];
+
+  dc.getFranklinRepos = getFranklinRepos;
+  /*dc.getDeployableRepos = getDeployableRepos;*/
+  dc.listFranklinRepos = listFranklinRepos;
+  /*dc.listDeployableRepos = listDeployableRepos;*/
+  dc.logout = logout;
+
+  dc.getFranklinRepos();
+
+  function getFranklinRepos() {
+    let repos = franklinAPIService.userRepos().getFranklinRepos();
+    repos.$promise.then(dc.listFranklinRepos);
+  };
+
+  /*function getDeployableRepos() {
+    let repos = franklinAPIService.userRepos().getDeployableRepos();
+    repos.$promise.then(dc.listDeployableRepos);
+  };*/
+
+  function listFranklinRepos(data) {
+    //TODO: push if it doesn't exits
+    dc.deployedRepos = [];
+    if (data.length > 0) {
+      dc.username = data[0].owner.name;
+      for (let repo of data) {
+        dc.deployedRepos.push({
+          "name": repo.name,
```

Comment:
Let's continue to use single quotes here. Also, the object keys don't need quotes unless they're dynamically computed. They seem to be static here.